docs/operating-scylla/procedures/backup-restore/backup.rst

```diff
@@ -29,11 +29,15 @@ With time, SSTables are compacted, but the hard link keeps a copy of each file.
 
 | 1. Data can only be restored from a snapshot of the table schema, where data exists in a backup. Backup your schema with the following command:
 
-| ``$: cqlsh -e "DESC SCHEMA" > <schema_name.cql>``
+| ``$: cqlsh -e "DESC SCHEMA WITH INTERNALS" > <schema_name.cql>``
 
 For example:
 
-| ``$: cqlsh -e "DESC SCHEMA" > db_schema.cql``
+| ``$: cqlsh -e "DESC SCHEMA WITH INTERNALS" > db_schema.cql``
+
+.. note::
+
+  To get proper schema description, you need to use cqlsh at least in version 6.0.19. To check version of your cqlsh, you can use ``cqlsh --version``.
```

Comment:
We should be very explicit here about the danger of not using the most recent version. In my opinion:

- This should be a warning,
- You should mention that using an older version of cqlsh might generate a schema description which does not contain all the details, and using it later to restore schema might result in incorrect schema and/or lead to data resurrection if it is followed by restoring data from a snapshot.

Reply:
This version isn't yet shipped with any scylla release

We didn't even pushed it to Scylla master 

So will need to backport to 6.0, but might miss it.

Reply:
It would get there eventually.

But no one is gonna hold a for release.

I wasn't even aware it's needed for 6.0

Reply:
Changed to warning and added information about data loss/resurrection